synapse/handlers/room.py

```diff
@@ -346,6 +347,31 @@ def clone_existing_room(
             if old_event:
                 initial_state[k] = old_event.content
 
+        # Resolve the minimum power level required to send any state event
+        # We will give the upgrading user this power level temporarily (if necessary) such that
+        # they are able to copy all of the state events over, then revert them back to their
+        # original power level afterwards
```

Comment:
Can you just add a quick comment saying that the temporary power levels here will get overridden later?